src/main/resources/templates/partials/scripts.html

```diff
@@ -10,9 +10,8 @@
     <body>
         <div th:fragment="scripts">
             <script src="/js/keys.js"></script>
-            <script src="https://code.jquery.com/jquery-3.5.1.js" integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=" crossorigin="anonymous"></script>
```

Comment:
Some pages may still need JQuery

Reply:
Changes made & pushed to donelle-media-responsive